common/src/test/java/org/apache/celeborn/common/network/TestHelper.java

```diff
@@ -30,4 +36,19 @@ public static CelebornConf updateCelebornConfWithMap(CelebornConf conf, Map<Stri
     }
     return conf;
   }
+
+  public static String getResourceAsAbsolutePath(String path) {
+    try {
+      File tempFile = File.createTempFile(new File(path).getName(), null);
+      tempFile.deleteOnExit();
+      URL url = SslSampleConfigs.class.getResource(path);
+      if (null == url) {
+        throw new IllegalArgumentException("Unable to find " + path);
+      }
+      FileUtils.copyInputStreamToFile(url.openStream(), tempFile);
+      return tempFile.getCanonicalPath();
+    } catch (IOException e) {
+      throw new RuntimeException("Failed to resolve path " + path, e);
+    }
+  }
```

Comment:
This is strictly not required for this PR - but keeps failing in `ApiBaseResourceAuthenticationSuite` (with maven) without it.